riotfile.py

```diff
@@ -1339,9 +1339,17 @@ def select_pys(min_version: str = MIN_PYTHON_VERSION, max_version: str = MAX_PYT
             },
             venvs=[
                 Venv(
-                    pys=select_pys(min_version="3.8"),
+                    pys=select_pys(min_version="3.9"),
                     pkgs={"botocore": "==1.34.49", "boto3": "==1.34.49"},
                 ),
+                Venv(
+                    pys=["3.8"],
+                    pkgs={"botocore": "==1.29.69", "boto3": "==1.26.69"},
```

Comment:
any reason these specific versions?

previously it was [~=1.13](https://pypi.org/project/botocore/1.13.50/) which says it should support Python 3.8


`~=1.13` was what was there before the change to `==1.34.49`.

or maybe more specifically, any reason why `==1.29.69` was chosen?

Reply:
It's two years old, which aligns with our version support policy if you don't interpret botocore's versioning as proper semver https://github.com/DataDog/dd-trace-py/blob/main/docs/contributing-integrations.rst#what-versions-of-a-given-library-should-an-integration-support